bedrock/firefox/views.py

```diff
@@ -284,7 +284,7 @@ class LatestFxView(TemplateView):
     """
     non_fx_redirect = 'firefox.new'
 
-    @vary_on_headers('User-Agent')
+    @cache_control(max_age=0)
```

Comment:
I considered using the never_cache decorator, but then out of curiosity I looked at the source and saw that it calls [patch_cache_control](https://github.com/django/django/blob/1f29164ced01bde92c72d20d67bc453484a4f078/django/utils/cache.py#L41-L87) which was not necessary for this case, so I used this method as an optimization. Given the traffic levels of this particular view, I felt the optimization was justified over the very slight loss of readability.